content/blog/2020-08-27-august-20-community-gems.md

```diff
@@ -0,0 +1,179 @@
+with `dvc repro` (`myfile is already trakced by SCM`).
+
+To get around this, you need to specify that DVC should not cache the outputs of
+your pipeline]. When you declare a pipeline stage with `dvc run`, instead of
+using the `-o` flag, use `-O`. This is shorthand for `--outs-no-cache` ([see
+docs for `dvc run` and its flags
+[here](https://dvc.org/doc/command-reference/run#options)).
+
+In your `dvc.yaml` file, if you prefer to manually edit, you'll add a field for
+`cache`:
+
+```yaml
+outs:
+  - myfile:
+      cache: false
+```
+
+### [Q: Can I change my `params.yaml` file to a `.json`?](https://discordapp.com/channels/485586884165107732/563406153334128681/730614265051873370)
+
+Yes, this is straightforward- you change your `params.yaml` to `params.json` in
+your workspace, and then use it in `dvc run`:
+
+```dvc
+$ dvc run -p params.json:myparam ...
+```
+
+For more about the `params.yaml` file,
+[see our docs](https://dvc.org/doc/start/experiments#defining-parameters).
+
+### [Q: Is there a guide for migrating from Git-LFS to DVC?](https://discordapp.com/channels/485586884165107732/485596304961962003/743559246599421974)
+
+We don't know of any published guide. One of our users shared their procedure
+for disabling LFS:
+
+```dvc
+$ git lfs uninstall
+$ git rm .gitattributes
+$ git rm .lfsconfig
+```
+
+Then you can `dvc add` files you wish to put in DVC tracking, and `dvc push`
+them to your remote. After that, `git commit` and you're good!
+
+Note that, if you're going to delete any LFS files, make sure you're certain the
+corresponding data has been transferred to DVC.
+
+### [Q: Is there a way to use DVC and CML to validate a model in a GitHub Action, without making the validation data available to the user opening the Pull Request?](https://discordapp.com/channels/485586884165107732/485596304961962003/739202123295883325)
+
+We don't have special support for this use case, and there may be some security
+downsides to using a confidential validation dataset with someone else's code
+(be sure nothing in their code could expose your data!). But, there are ways to
+implement this if you're sure about it.
+
+We'd recommend creating a separate "data registry" repository using a private
+cloud bucket to store your validation dataset
+([see our docs about the why and how of data registries](https://dvc.org/doc/use-cases/data-registries#data-registries)).
+Your CI system can be setup to have access to the data registry via secrets
+(called "variables" in GitLab). Then when you run validation via
+`dvc repro validate`, you could use `dvc get` to pull the private data from the
+registry.
+
+The data is never exposed to the user in an interactive setting, only on the
+runner- and there it's ephemeral, meaning it does not exist once the runner
+shuts down.
+
+## CML questions
+
+### Q: Sometimes when I make a commit on a branch, my CI workflow isn't triggered. What's going on?
```

Comment:
is there a link here?

Reply:
it's from a YouTube comment, but YouTube doesn't let you link to a comment. Only the video itself.

Reply:
https://www.youtube.com/watch?v=PnmfkLiMLHs ?

Reply:
Giving this method a try